src/structures/interfaces/TextBasedChannel.js

```diff
@@ -244,11 +244,6 @@ class TextBasedChannel {
    * Messages or number of messages to delete
    * @param {boolean} [filterOld=false] Filter messages to remove those which are older than two weeks automatically
    * @returns {Promise<Collection<Snowflake, Message>>} Deleted messages
-   * @example
```

Comment:
Why was this example removed?

Reply:
cuz you suck

Reply:
That was a mess-up while merging the recent master changes. (I do believe I had an example there before), thank you for telling